twill-common/src/main/java/org/apache/twill/filesystem/LocalLocation.java

```diff
@@ -251,6 +251,34 @@ public boolean mkdirs() throws IOException {
     return file.mkdirs();
   }
 
+  @Override
+  public boolean mkdirs(String permission) throws IOException {
```

Comment:
I see, in that case I think it would be good to add this to the javadoc of the method.